inference-engine/tests/functional/plugin/gpu/shared_tests_instances/low_precision_transformations/strided_slice_transformation.cpp

```diff
@@ -86,7 +86,7 @@ const std::vector<LayerTestsDefinitions::StridedSliceTransformationParam> params
     }
 };
 
-INSTANTIATE_TEST_CASE_P(smoke_LPT, StridedSliceTransformation,
+INSTANTIATE_TEST_CASE_P(DISABLED_smoke_LPT, StridedSliceTransformation,
```

Comment:
Please use `skip_test_config` mechanism instead of `DISABLED_` prefix. CC @vladimir-paramuzov

Reply:
This will be fixed in PR #4016